src/validator.js

```diff
@@ -36,6 +37,7 @@ export default class Validator {
     this.chalk = new chalk.constructor(
       {enabled: !this.config.boring});
     this.collector = new Collector();
+    this.addonMetaData = {};
```

Comment:
I wonder if this should be null, then we could more easily do a check and return it if it's already defined. Can be a todo or new issue.